workspace/utilities/lib/share-link.xsl

```diff
@@ -45,7 +45,7 @@
 		
 		<xsl:variable name="computed-attr">
 			<xsl:copy-of select="$attr" />
-			<xsl:add data-mailto="{$computed-url}" />
+			<add data-mailto="{$computed-url}" />
```

Comment:
This should have been

1. A seperate PR
1. a `set` ?

Reply:
Change add to set for sure

Reply:
Ping @beaubbe